packages/eslint-config-mad/README.md

```diff
@@ -0,0 +1,29 @@
+# Equinor MAD ESLint
+
+This package includes all ESLint configurations used in the MAD Team at Equinor.
+
+## Config structure
+
+The main package consists of multiple entrypoints:
+
+-   `base`
+-   `react`
+-   `typescript`
+
```

Comment:
Looks like a good start! So if I understand it correctly, `base` rules that fit any javascript project, `typescript` has rules that fit any typescript project, and `react` any react project, correct?
And if you use `@equinor/mad`, you use all rules?

I would suggest having one for react-native as well. Should we create an issue for it?

Reply:
And I guess the goal is to have two sets of rules that our projects would use: `@equinor/mad/web`, which extends `base`, `react`, `typescript` and maybe a web-specific ruleset, and `@equinor/mad/mobile` which would extend `base`, `react`, `typescript` and some mobile specific ruleset (react-native). Am I correct?

Reply:
That is correct.
This should be a good starting point for our projects I think. 
As we keep on arguing over details in PR's, I think we should make feature requests to this package and let it grow organically. 
As time passes, I hope the best structure will emerge.